pkg/resourceinterpreter/configurableinterpreter/configmanager/accessor.go

```diff
@@ -8,8 +8,8 @@ import (
 
 // CustomConfiguration provides base information about custom interpreter configuration
 type CustomConfiguration interface {
-	Name() string
 	TargetResource() schema.GroupVersionKind
+	MergeCustomizationRules(rules configv1alpha1.CustomizationRules)
```

Comment:
```suggestion
	Merge(rules configv1alpha1.CustomizationRules)
```
We can shorten the function name here as the parameter gives the extra info about what it is going to merge.